README.md

```diff
@@ -28,24 +28,31 @@ $ make
 
 This will attempt to clone the repositories into ``SDX_HOME`` and run their ``Dockerfile``'s
 
-Once built, you can bring the services up with:
+To git pull all the sdx repos (and to discover what branch each repo is on):
 
 ```shell
-$ make start
+$ make update
 ```
 
-Rebuild services:
+To rebuild the services:
 
 ```shell
 $ make build
 ```
 
-To git pull all the sdx repos (and to discover what branch each repo is on):
+Once built, you can bring the services up with:
 
 ```shell
-$ make update
+$ make start
 ```
 
+Once the services have started you can log in at localhost with:
```

Comment:
One small change here. This can be something like:
Once the services have started, you can access sdx-console on `http://localhost`.  sdx-console is an easy way to test the features of SDX.  The login credentials for this service are:

Reply:
Updated to include your suggested changes